configuration/backend_configuration/ampathforms/procedure-history-form.json

```
@@ -0,0 +1,174 @@
+                  },
+                  {
+                    "concept": "160720AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
+                    "label": "Procedure outcome unsatisfactory to physician"
+                  }
+                ]
+              },
+              "id": "procedureOutcome",
+              "label": "Procedure outcome"
+            },
+            {
+              "type": "obs",
+              "questionOptions": {
+                "rendering": "number",
+                "concept": "165929AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
+                "min": "0",
+                "max": "100"
+              },
+              "id": "procedureDuration",
+              "label": "Duration of procedure"
+            },
+            {
+              "type": "obs",
+              "questionOptions": {
+                "rendering": "select",
+                "concept": "1732AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
+                "answers": [
+                  {
+                    "concept": "1072AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
+                    "label": "Days"
+                  },
+                  {
+                    "concept": "1822AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
+                    "label": "Hours"
+                  },
+                  {
+                    "concept": "1733AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
+                    "label": "Minutes"
+                  },
+                  {
+                    "concept": "1074AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
+                    "label": "Months"
+                  },
+                  {
+                    "concept": "162582AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
+                    "label": "Number of occurrences"
+                  },
+                  {
+                    "concept": "162583AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
+                    "label": "Seconds"
+                  },
+                  {
+                    "concept": "1073AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
+                    "label": "Weeks"
+                  },
+                  {
+                    "concept": "1734AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
+                    "label": "Years"
+                  }
+                ]
+              },
+              "id": "durationUnit",
+              "label": "Duration units"
+            },
+            {
+              "type": "obs",
+              "questionOptions": {
+                "rendering": "textarea",
+                "concept": "160716AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
+                "rows": 5
+              },
+              "id": "procedureComment",
+              "label": "Procedure comment"
+            }
+          ]
+        }
+      ]
+    }
+  ],
+  "processor": "EncounterFormProcessor",
+  "encounter": "Procedure History",
+  "encounterType": "6f8393e7-cf3d-45e1-abc1-d83858a18d8c",
+  "referencedForms": [],
+  "uuid": "4965333c-6a8d-4f32-8030-9abf57b62058",
+  "description": "Encounter type for Procedure History",
```

```suggestion
  "description": "Procedure History Form - used for documenting completed medical procedures",
```

Or something to that effect.